terraform/ec2/win/main.tf

```diff
@@ -190,12 +190,24 @@ resource "null_resource" "integration_test_run_validator" {
     destination = module.validator.instance_validator_config
   }
 
+  //runs validator and sets up prometheus java agent
   provisioner "remote-exec" {
     inline = [
+      "mkdir C:\\jmx_workload",
+      "powershell.exe -Command \"'---', 'rules:', '- pattern: \\\".*\\\"' | Set-Content -Path \\\"C:\\jmx_workload\\exporter_config.yaml\\\"\"",
+      "powershell.exe -Command \"'global:', '  scrape_interval: 1m', '  scrape_timeout: 10s', 'scrape_configs:', '  - job_name: jmx-exporter', '    sample_limit: 10000', '    file_sd_configs:', '      - files: [ \\\"C:\\\\jmx_workload\\\\prometheus_file_sd.yaml\\\" ]' | Set-Content -Path \\\"C:\\jmx_workload\\prometheus.yaml\\\"\"",
+      "powershell.exe -Command \"'- targets:', '  - 127.0.0.1:9404', '  labels:', '    application: test-app', '    InstanceId: ${var.ec2_instance_type}' | Set-Content -Path \\\"C:\\jmx_workload\\prometheus_file_sd.yaml\\\"\"",
```

Comment:
nit: This doesnt have to be in a separate sd file... can just inline as static targets.